pkg/apis/flagger/v1beta1/canary.go

```diff
@@ -66,6 +66,10 @@ type CanarySpec struct {
 	// +optional
 	MetricsServer string `json:"metricsServer,omitempty"`
 
+	// InsecureSkipVerify disables certificate verification for the provider
+	// +optional
+	InsecureSkipVerify bool `json:"insecureSkipVerify,omitempty"`
```

Comment:
Please add this to the CRD in https://github.com/fluxcd/flagger/blob/main/artifacts/flagger/crd.yaml#L1071 then run `make crd` and commit all changes.

Reply:
fixed!